pkg/uhttp/transport.go

```diff
@@ -100,8 +101,16 @@ func (uat *debugTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	reg, err := regexp.Compile("Authorization: Bearer .*\n")
```

Comment:
yeah dont compile inline func, but sure, hack?

Reply:
I put it in an init block. "hack" or not, it's a good way to remove bearer tokens from the debug output